example/App.js

```diff
@@ -33,17 +33,19 @@ const matomoConfig =
     ? {
         baseUrl: 'https://demo2.matomo.org/piwik.php',
         siteId: 23,
+        contentBase: 'https://demo.matomo.org',
```

Comment:
Here I've attempted to mimic the behaviour of the example applications for [iOS](https://github.com/matomo-org/matomo-sdk-ios/tree/develop/Example) and [Android](https://github.com/matomo-org/matomo-sdk-android/tree/master/exampleapp) as closely as possible. Changing the content base might interfere with the metrics collected in the matomo demo server.